.gitignore

```diff
@@ -1,4 +1,8 @@
 *.swp
 *~
 _output
+__debug_bin
+apiserver.local.config/
 .idea/
+.vscode/
+*.out
```

Comment:
Let's not had files that are not generated by the project to the .gitignore. For any file related to your IDE or testing env, it is better to have a global gitignore on your machine.

Reply:
Sure.